tasks/kibana.yml

```diff
@@ -26,23 +34,6 @@
     - reload systemd
     - restart kibana
 
-- name: Extract Kibana tarball.
-  unarchive:
-    copy: no
-    src: /opt/kibana-{{ elk_kibana_version }}-linux-x64.tar.gz
-    dest: /opt/
-    owner: "{{ elk_nginx_user }}"
-    group: "{{ elk_nginx_user }}"
-  when: kibana_directory_check.stat.exists == false
-
-- name: Symlink Kibana directory.
-  file:
-    state: link
-    dest: /opt/kibana
-    src: /opt/kibana-{{ elk_kibana_version }}-linux-x64
-    owner: "{{ elk_nginx_user }}"
-    group: "{{ elk_nginx_user }}"
-
   # Two-pass approach to creating logfile idempotently. Create the empty logfile
   # if none exists (force: no), but to ensure permissions we need a second task.
 - name: Create logfile for Kibana.
```

Comment:
Does the deb package preinst/postinst script handle creating a log file? If so, maybe we can collapse the two-pass approach into a single task to enforce permissions. Worth looking at, but leaving it alone should be fine too.